app/MMB/src/scenes/timeline/events/LeaveForAirportTimelineEvent.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+
+import * as React from 'react';
+import idx from 'idx';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import {
+  Translation,
+  DateFormatter,
+  TranslationFragment,
+} from '@kiwicom/mobile-localization';
+import { Text, Color } from '@kiwicom/mobile-shared';
+
+import type { LeaveForAirportTimelineEvent as LeaveForAirportTimelineEventType } from './__generated__/LeaveForAirportTimelineEvent.graphql';
+import TimelineEvent from '../TimelineEvent';
+import TimelineEventIcon from '../TimelineEventIcon';
+import DateLocation from '../TimelineEventDateLocation';
+
+type Props = {|
+  +data: LeaveForAirportTimelineEventType,
+|};
+
+const LeaveForAirportTimelineEvent = (props: Props) => {
+  const timestamp = idx(props, _ => _.data.timestamp);
+  let time = null;
+  let isPastEvent = false;
+  if (timestamp) {
+    time = DateFormatter(new Date(timestamp)).formatToTime();
+    isPastEvent = new Date() - new Date(timestamp) > 0;
+  }
+  let dateLocation = null;
+  if (time) {
+    dateLocation = <DateLocation time={time} />;
+  }
+
+  return (
+    <TimelineEvent
+      isPastEvent={isPastEvent}
+      dateLocation={dateLocation}
+      iconVertLines={
+        <TimelineEventIcon name="directions-walk" isPastEvent={isPastEvent} />
+      }
+      mainContent={
+        <TranslationFragment>
+          <Translation id="mmb.booking_timeline.event.leave_for_airport.title" />
+          <Text style={{ fontSize: 13, color: Color.ink.light }}>
```

Comment:
Please try to avoid inline styles. This object is created every time and it may cause problems with `PureComponent` later (`Text` is one of the adepts for `PureComponent`)...

Reply:
I will use StyleSheet. I had thought about it and then forgot. Sorry about that.